src/main/java/duke/Task.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+public class Task {
+    public String getSYMBOL;
```

Comment:
Consider adhering to camel case (e.g. `getSymbol`) as mentioned in the coding standards, to improve consistency.